lib/teaspoon/suite.rb

```diff
@@ -94,8 +94,10 @@ def asset_from_file(original)
       normalize_js_extension(filename)
     end
 
-    def normalize_js_extension(filename)
-      filename.gsub(".erb", "").gsub(/(\.js\.coffee|\.coffee|\.es6|\.js\.es6|\.es6\.js)$/, ".js")
+    def normalize_js_extension(original_filename)
```

Comment:
If I recall it was not public and involved a bunch of complexity around tilt or something to determine what extensions are understood, and I think maybe wanted to pass the file off at that point to be processed. So not super useful there, but I'm not 100% sure. The config is only to try and find spec files (if I recall), and that's through the file system via a glob call. Might be worth code diving to see if that's improved since last time though.

---
Jeremy Jackson

> On Oct 8, 2015, at 6:42 PM, Mike Pack <notifications@github.com> wrote:
> 
> In lib/teaspoon/suite.rb:
> 
> > @@ -94,8 +94,10 @@ def asset_from_file(original)
> >        normalize_js_extension(filename)
> >      end
> >  
> > -    def normalize_js_extension(filename)
> > -      filename.gsub(".erb", "").gsub(/(\.js\.coffee|\.coffee|\.es6|\.js\.es6|\.es6\.js)$/, ".js")
> > +    def normalize_js_extension(original_filename)
> @jejacks0n I'll look tomorrow but do you know offhand if sprockets can do this normalization for us? Would be nice.
> 
> —
> Reply to this email directly or view it on GitHub.
>